src/components/MarkdownViewer.tsx

```
@@ -0,0 +1,317 @@
+            /* ---------- analyse ---------- */
+            const pageRisks = await analyzeTextWithAI(organizedText, sectionNumber);
+            if (pageRisks.length) {
+              useMouStore.getState().setRiskyClauses(pageRisks);
+              setRisks(prev => [...prev, ...pageRisks]);
+            }
+    
+            setProcessedPages(prev => [...prev, sectionNumber]); // progress bar
+          })
+        );
+    
+        await Promise.all(tasks); // max 3 running simultaneously
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)